src/client/src/App.tsx

```diff
@@ -141,7 +129,9 @@ const App: React.FC = () => {
         );
         for (const e of sortedElementsToRemove) {
             if (isNode(e)) {
-                await nodeService.deleteNode(e);
+                await nodeService
+                    .deleteNode(e)
+                    .catch((e: Error) => console.log('Error deleting Node', e));
```

Comment:
I would use a try catch block here instead of .catch(), since you are using 'await' and it will be easier to expand this part later.